src/main/java/octo/waddle/hellodropwizard/api/Message.java

```diff
@@ -0,0 +1,18 @@
+package octo.waddle.hellodropwizard.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import org.hibernate.validator.constraints.Length;
+
+public class Message{
+
+    @Getter
+    private String content;
```

Comment:
jeżeli content się nie zmiania , a tutaj też nie ma jak się zmienić oznacz ten field jako `final`
`immutable vs mutable` <- poszukaj i poczytaj
Ogólnie używaj immutable wszędzie gdzie możesz.
Wtedy jeżeli bezargumentowy konstruktor jest nie używany to go usuń,
usuń Getter'a z fielda i dodaj `@Value` na klasie

Cała klasa wtedy będzie wyglądać tak:
```
@Value
public class Message {
  private final String content;
}
```